unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/ComponentID.cs

```diff
@@ -6,14 +6,13 @@ public static class ComponentID
         public const int ANIMATOR = 1042;
         public const int AUDIO_SOURCE = 1020;
         public const int AUDIO_STREAM = 1021;
+        public const int AVATAR_ATTACH = 1070;
         public const int BOX_SHAPE = 1013;
         public const int CYLINDER_SHAPE = 1016;
-        public const int G_L_T_F_SHAPE = 1040;
+        public const int G_L_T_F_SHAPE = 1041;
         public const int N_F_T_SHAPE = 1040;
```

Comment:
Is this autogenerated? can we use GLTF_SHAPE and NFT_SHAPE?

Reply:
Yes it is autogenerated, I already fixed it in another PR where it will change the name at generation time